ping-pong-game/index.html

```diff
@@ -38,44 +38,65 @@
             
         })
 
-        
+        var value1 = 0;
 
```

Comment:
Put a comment that
1. moveUp1 is for right side plate
2. moveup2 is for left side plate